### ISSUE TITLE
Hi Griffpatch

### ISSUE THREAD

**Issue by Secret-chest** [griffpatch/griffpatch.github.io#32]
This is a more relaxed place, isn't it?
I just wanna say I love your work, both your Scratch games as well as Scratch Addons. You truly made Scratch a better place for all. You deserve every follow.

**Comment by SherryTheKitten** [comment]
I agree @Secret-chest. I am so unpossessed with @griffpatch work/projects.

**Comment by SherryTheKitten** [comment]
@Secret-chest How do you open at chat thing like this?

**Comment by Secret-chest** [comment]
@SherryTheKitten It's called an issue. Click new issue

**Comment by EngineerRunner** [comment]
this isn't what the issues page is for.
